src/packagepath.ts

```diff
@@ -17,10 +16,9 @@ export async function getPkgPath() {
         const res = await execFile(
             juliaExecutable.file,
             [
-                '--startup-file=no',
                 '--history-file=no',
                 '-e',
-                'using Pkg; println(Pkg.depots()[1])'
+                'using Pkg;haskey(ENV, "JULIA_PKG_DEVDIR") ? println(ENV["JULIA_PKG_DEVDIR"]) : println(joinpath(Pkg.depots()[1], "dev"))'
```

Comment:
```suggestion
                'using Pkg;println(get(ENV, "JULIA_PKG_DEVDIR", joinpath(Pkg.depots()[1], "dev")))'
```

Here is a slightly more succinct version :) But also happy to merge the original one.

Reply:
I was wondering how I could idiomatically retrieve the values from the environment, but I couldn't find it anymore. Thanks!